lib/packages/docutils/rstgen.nim

```diff
@@ -57,6 +57,8 @@ type
     options*: RstParseOptions
     findFile*: FindFileHandler
     msgHandler*: MsgHandler
+    destBaseDir*: AbsoluteDir
+    destFile*: AbsoluteFile
     filename*: string
```

Comment:
maybe add doc comments to explain what `filename` is (i'm guessing it's the source file)

Reply:
yes.. it was already there though. I'll still document it.

Reply:
Done.